duckduckgo/duckduckgo-privacy-extension
# Add animation for de-whitelisting a site

**Reviewer:** @andrey-p 



## Description:
Following up from previous work in #158, also add an animation for when a site is removed from the whitelist via the main popup toggle.
The animation behaves exactly like the "Added to Whitelist" one, but the text is "Removed from Whitelist"

## Steps to test this PR:
1. Build and reload extension
2. Try whitelisting a site - a brief transition showing the text "Added to Whitelist" should appear and then tab should reload
3. Try de-whitelisting a site - the same transition should appear, but with the "Removed to Whitelist" text instead, and a shield icon


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
